### PR TITLE
Add verbose

### PR DESCRIPTION
**Pull request by Agrendalath**
Added verbose mode referenced in #3.
I did not write test for this one, because packages during testing process have no external dependencies.

### CODE REVIEW

File: pip_chill/pip_chill.py [hunk 1]
```diff
@@ -3,25 +3,42 @@
 
 import pip
 
+from utils import Distribution
```
Comment (lsh-0) [inline pip_chill/pip_chill.py]: I had to make this `from .utils import Distribution` to make `pip-chill -v` work

Comment (rbanffy) [reply pip_chill/pip_chill.py]: Sorry... Only saw this now.

**Comment by rbanffy** [comment]
Adding both of you, @Agrendalath and @lsh-0 to the AUTHORS file.

**Comment by lsh-0** [comment]
cheers @rbanffy . I did some experimental hacking on a fork of this branch here to ensure that editable vcs entries were preserved: https://github.com/elifesciences/pip-chill/commit/5ca9c5fd53ae6d547c6d397aac1fad2a4f4a0805

but it was some months ago and I don't recall the final outcome and you may have fixed it since. feel free to take a look and merge/discard as you wish.

**Pull request merged**